flujo/validation/fixers.py

```
@@ -0,0 +1,430 @@
+        )
+    except Exception:
+        return 0
+
+
+# --- V-T1 Fixer implementation ---
+
+
+def _vt1_preview(file_path: str, report: Any) -> int:
+    try:
+        text = Path(file_path).read_text(encoding="utf-8")
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)